sklearn/preprocessing/imputation.py

```diff
@@ -142,27 +147,36 @@ def fit(self, X, y=None):
                              " got strategy={1}".format(allowed_strategies,
                                                         self.strategy))
 
-        if self.axis not in [0, 1]:
+        if self.axis is None:
+            self._axis = 0
+        else:
+            warnings.warn("Parameter 'axis' has been deprecated in 0.20 and "
+                          "will be removed in 0.22. Future (and default) "
+                          "behavior is equivalent to 'axis=0' (impute along "
+                          "columns).", DeprecationWarning)
```

Comment:
We should probably say "Row-wise imputation can be performed with FunctionTransformer." Could even specify `FunctionTransformer(lambda X: Imputer(...).fit_transform(X.T).T)`